Salaried teacher training - adding info on fees (#4421)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support/salaried-teacher-training.md

There are some teacher training courses that come with a salary. 

This means you will get paid a salary while you train to get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). You'll need QTS to teach in many primary, secondary and special schools in England. 

> Some salaried teacher training courses may also come with fees, check with individual provders for more information. 

Reviewer: ```suggestion
Some salaried teacher training courses may also come with fees, Check with individual providers for more information. 
```

(Am aware that disliking commas like this is a me thing so please ignore if needed)

> The main salaried courses include:

Reviewer: Do we need `The main`?

> The main salaried courses include:

Author: ```suggestion
The salaried courses are:
```

* School Direct salaried
* postgraduate teaching apprenticeships (PGTA)

> * Teach First teacher training

Author: ```suggestion
* Teach First teacher training

If you do not already have a degree, you maybe eligible for a [teacher degree apprenticeship (TDA)](/train-to-be-a-teacher/teacher-degree-apprenticeships).
```

There are some teacher training courses that come with a salary. 

This means you'll be paid a salary while you train to get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). You'll need QTS to teach in many primary, secondary and special schools in England. 

Some salaried teacher training courses may also come with fees. Check with individual providers for more information. 

> The salaried courses are:

Reviewer: ```suggestion
The postgraduate salaried courses are:
```
Is it worth adding `postgraduate` as we are not including TDA on this page?

* School Direct salaried
* postgraduate teaching apprenticeships (PGTA)
* Teach First teacher training

> If you do not already have a degree, you maybe eligible for a [teacher degree apprenticeship (TDA)](/train-to-be-a-teacher/teacher-degree-apprenticeships).

Author: I can't get a grip on my 'maybes' on this PR!

I am also in 2 minds, thought it might be useful if people are browsing and want to know if there's a salaried undergrad course? Equally happy to remove it though!

> If you do not already have a degree, you may be eligible for a [teacher degree apprenticeship (TDA)](/train-to-be-a-teacher/teacher-degree-apprenticeships). A TDA is an undergraduate courses where you’ll work in a school and earn a salary while getting a degree and QTS.

Author: ```suggestion
If you do not already have a degree, you may be eligible for a [teacher degree apprenticeship (TDA)](/train-to-be-a-teacher/teacher-degree-apprenticeships). A TDA is an undergraduate teacher training course where you’ll work in a school and earn a salary while getting a degree and QTS.
```

> If you do not already have a degree, you may be eligible for a [teacher degree apprenticeship (TDA)](/train-to-be-a-teacher/teacher-degree-apprenticeships). A TDA is an undergraduate teacher training course where you’ll work in a school and earn a salary while getting a degree and QTS.

Reviewer: Is it worth moving this to the `Am I eligible section`, perhaps under an h3 of `If you don't have a degree` or similar?

All salaried courses are broadly the same, but the course length and qualifications you work towards may vary.

These courses are in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible. 

> ## Am I eligible for a salaried teacher training course? 

Reviewer: ```suggestion
## Am I eligible for a postgraduate salaried teacher training course? 
```
Worth adding `postgraduate` to the h2?

## How much will I be paid on a salaried course? 

> You'll receive an [unqualified teacher’s salary](/life-as-a-teacher/pay-and-benefits/teacher-pay), the exact amount will vary depending on the school you work in. 

Reviewer: ```suggestion
You'll receive an [unqualified teacher’s salary](/life-as-a-teacher/pay-and-benefits/teacher-pay). The exact amount will vary depending on the school you work in. 
```

## How does salaried teacher training work? 

Salaried teacher training is usually similar to non-salaried teacher training. You'll spend most of your time in school placements, with some theoretical learning. 

$nathan$

> You will not perform more than 90% of a full-time teacher’s duties on most salaried teacher training courses. If you are on a postgraduate teaching apprenticeship, you'll perform no more than 80% of a full-time teacher's duties. 

Author: ```suggestion
You will not perform more than 90% of a full-time teacher’s duties on most salaried teacher training courses. If you're on a postgraduate teaching apprenticeship, you'll perform no more than 80% of a full-time teacher's duties. 
```

## How does salaried teacher training work? 

Salaried teacher training is usually similar to non-salaried teacher training. You'll spend most of your time in school placements, with some theoretical learning. 

$nathan$

> You will not perform more than 90% of a full-time teacher’s duties on most salaried teacher training courses. If you're on a postgraduate teaching apprenticeship, you'll perform no more than 80% of a full-time teacher's duties. 

Reviewer: ```suggestion
You will not perform more than 90% of a full-time teacher’s duties on most salaried teacher training courses. If you're doing a postgraduate teaching apprenticeship, you'll perform no more than 80% of a full-time teacher's duties. 
```
Not sure what the right wording is!
